### ISSUE TITLE
How to import 

### ISSUE THREAD

**Issue by jamnabai** [snipe/snipe-it#6679]
![snipit user import](https://user-images.githubusercontent.com/42636964/52205086-fcedde00-289b-11e9-86d5-f7ab6efeb9f2.PNG)

Some issue in user import plz help how to import users around 400 users

**Comment by jamnabai** [comment]
Hello Team,

 

 

      This mail in regards to the importing of users , we are trying from past 3 days but unable to import the file as it shows us the status as importing and get stuck, we have around 600 users.

 

We took the format from your site for importing and made the data accordingly but still unable to import. 

We also tried to import one user from the give format but still it get stuck and unable to import the format is same as of yours.

Also attaching the file for your reference.

 

Kindly help us to do the needful as we found the freeware very useful for asset management.

 

 

 

Best Regards,
 
Afzal Momin
-----------------------------------------------------------
Narsee Monjee Educational Trust's
Jamnabai Narsee School,
Mumbai - India.
Website:  <http://www.jns.ac.in> www.jns.ac.in
Tel: +91 22 26187575 / 7676 , Ext(168)
Cell: +91  8879068884
----------------------------------------------------------
